### ISSUE TITLE
"load the public SARS-CoV-2 tree" option loads old tree

### ISSUE THREAD

**Issue by yatisht** [theosanderson/taxodium#67]
Hey Theo (cc  @AngieHinrichs),

I think there might be an issue in updating the `nodelist.pb` with the latest public tree release on https://taxodium.org/. As a result, the "load the public SARS-CoV-2 tree" ends up loading an old tree release.  Would it be easier to simply load the taxodium protobufs that we're sharing now with each release (e.g. https://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2/2021/09/21/public-2021-09-21.all.masked.taxodium.pb.gz)? 

-Yatish

**Comment by theosanderson** [comment]
Thanks Yatish,

Sorry about this, I think it had started running out of RAM and I've kept meaning to fix it but not quite managing it. I've changed it to use https://hgwdev.gi.ucsc.edu/~angie/UShER_SARS-CoV-2/public-latest.all.masked.taxodium.pb.gz . As a heads up, there's a likelihood I may revert this to something I make eventually once I fix it because I want to keep the latitude to be able to change things up by e.g. adding nucleotide mutations. Apologies again for the delay on this and thanks for flagging!

-T
